src/main/scala/cats/effect/bio/BIO.scala

```diff
@@ -188,7 +188,7 @@ sealed abstract class BIO[E, +A] {
    * @see [[runCancelable]] for the version that gives you a cancelable
    *      token that can be used to send a cancel signal
    */
-  final def runAsync(cb: Either[E, A] => BIO[E, Unit]): BIO[Throwable, Unit] = BIO {
+  final def runAsync[E1 >: E](cb: Either[E, A] => BIO[Throwable, Unit]): BIO[Throwable, Unit] = BIO {
```

Comment:
`E1` seems unused here and in `runCancelable`, a mistake perhaps? :)

Reply:
good catch!